lib/rspec-gc-control/base_text_formatter.rb

```diff
@@ -1,45 +1,50 @@
+require 'rspec/core/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.